includes/class-wc-payments-upe-split-blocks-payment-method.php

```diff
@@ -17,13 +17,8 @@ class WC_Payments_UPE_Split_Blocks_Payment_Method extends WC_Payments_Blocks_Pay
 	public function get_payment_method_script_handles() {
 		$classic_blocks_scripts = parent::get_payment_method_script_handles();
 
-		wp_register_script(
-			'WCPAY_BLOCKS_UPE_SPLIT_CHECKOUT',
-			plugins_url( 'dist/upe-split-blocks-checkout.js', WCPAY_PLUGIN_FILE ),
-			[ 'stripe' ],
-			'1.0.1',
-			true
-		);
+		WC_Payments::load_script_with_dependencies( 'WCPAY_BLOCKS_UPE_SPLIT_CHECKOUT', 'dist/upe-split-blocks-checkout', [ 'stripe' ], true, '1.0.1' );
```

Comment:
I believe that the `1.0.1` version here was only added, because it's a required parameter, and someone wanted to get to `in_footer = true`. As I mentioned in another comment, let's remove the footer parameter altogether, and then neither the version, nor the boolean would be needed here.

Reply:
Fixed in 8c9a994cce0c7154f43e9cd005e9bbba0f99f7dd